src/ProxyClient/Http/InvalidationRequest.php

```diff
@@ -21,7 +21,7 @@
 class InvalidationRequest implements RequestInterface
 {
     use RequestDecorator;
```

Comment:
Ah, I see. AFAIK message decorator was never required by anything, but I could be wrong.